### ISSUE TITLE
Fast xtts implementation

### ISSUE THREAD

**Issue by cduk** [matatonic/openedai-speech#80]
You might be interested to look at this:

https://github.com/astramind-ai/Auralis/tree/main

**Comment by bi1101** [comment]
Agree, the vLLM back end and the optimization offers pretty good speed up and concurrency.

**Comment by matatonic** [comment]
I saw this as well, it's on my radar but haven't looked at it yet. Seems promising!

**Comment by maxi1134** [comment]
This would be great for those of us using live answers like in Home-Assistant!

**Comment by matatonic** [comment]
Sorry I haven't updated this in a while, I still haven't found the time to get back to working on it again...

**Comment by matatonic** [comment]
astramind-ai/Auralis has an openai API now, so I'd recommend anyone looking for a better server to try that.
